docs/developers/contributing.rst

```diff
@@ -209,11 +179,11 @@ the ``--no-verify`` option with ``git commit``.
 Test coverage
 ~~~~~~~~~~~~~
 
-Zarr maintains 100% test coverage under the latest Python stable release (currently
-Python 3.8). Both unit tests and docstring doctests are included when computing
+Zarr strives to maintain 100% test coverage under the latest Python stable release
+(currently Python 3.12). Both unit tests and docstring doctests are included when computing
```

Comment:
```suggestion
Both unit tests and docstring doctests are included when computing
```
Latest stable Python is 3.13, but I'd just get rid of this to save having to update it every new release